src/gc/gc.h

```diff
@@ -108,6 +108,7 @@ extern "C" uint8_t* g_gc_highest_address;
 extern "C" bool g_fFinalizerRunOnShutDown;
 extern "C" bool g_built_with_svr_gc;
 extern "C" uint8_t g_build_variant;
+extern "C" VOLATILE(int32_t) g_no_gc_lock;
```

Comment:
>g_no_gc_lock [](start = 29, length = 12)

this should either just be in gc.cpp or in gcimpl.h. it's an pure internal implementation.